### DISCUSSION TITLE
Not scraping benign looking sites/pages

### DISCUSSION THREAD

**Discussion by hragbalian** [spider-rs/spider#189] [Q&A]
Hi there, 

I'm encountering a number of sites that Spider won't scrape.  Here's a random example: https://www.koreaboo.com/news/yedam-onepact-armada-entertainment/.  I've tried enabling the proxy and the anti_bot parameters to no avail, even through the scrape interface in the console. 

Any help would be greatly appreciated.

**Comment by evanokeefe39** [comment]
Is the service down? I am also not getting any documents from crawls that were previously producing hundreds of docs...

**Reply by j-mendez** [reply]
> Is the service down? I am also not getting any documents from crawls that were previously producing hundreds of docs...

Crawls should work again. Saw an issue this morning in a crate that we use for spider, looks like the crate has an area that is calling unwrap on an error. Simple fix that should be out today.
